packages/protocol/socket.ts

```diff
@@ -121,21 +121,18 @@ const noCallerStackTracesForFailedCommands = new Set<CommandMethods>([
 ]);
 
 export type ExperimentalSettings = {
-  listenForMetrics: boolean;
   controllerKey?: string;
   disableCache?: boolean;
+  disableConcurrentControllerLoading?: boolean;
+  disableIncrementalSnapshots?: boolean;
+  disableProtocolQueryCache?: boolean;
+  disableRecordingAssetsInDatabase?: boolean;
   disableScanDataCache?: boolean;
-  disableQueryCache?: boolean;
-  disableStableQueryCache?: boolean;
-  disableUnstableQueryCache?: boolean;
   enableRoutines?: boolean;
-  rerunRoutines?: boolean;
-  profileWorkerThreads?: boolean;
-  disableRecordingAssetsInDatabase?: boolean;
   keepAllTraces?: boolean;
-  disableIncrementalSnapshots?: boolean;
-  disableConcurrentControllerLoading?: boolean;
-  disableProtocolQueryCache?: boolean;
+  listenForMetrics: boolean;
```

Comment:
Sorry there is a bunch of diff noise here --- I wanted to sort the lines because it was bothering me.

Reply:
yay 🥳  I often do the same, no worries